Sprint-3/todo-list/script.js

```diff
@@ -1,25 +1,97 @@
+const toDoInput = document.getElementById("inputToDo");
+const addToDoBtn = document.getElementById("addToDo");
+const container = document.getElementById("todo-list");
```

Comment:
I can't find any use of the variable `container` in your code.

Reply:
Sorry, I needed to refactor my codes.
Thanks for this comment